lib/presentation/widgets/about_the_app_presentation_widgets/expandable_page_view.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+
+class ExpandablePageView extends StatefulWidget {
```

Comment:
Review the constants and trailing commas in this file

Reply:
I missed a lot of commas, sorry. Fixed :)